framework/Models/PostProcessors/Validation.py

```diff
@@ -0,0 +1,256 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+  Created on March 20, 2021
+  @author: alfoa
+  description: Postprocessor named Validation. This postprocessor is aimed to
+               to represent a gate for any validation tecniques and processes
+"""
+
+#External Modules---------------------------------------------------------------
+import numpy as np
+import copy
+import time
+import xarray as xr
+#External Modules End-----------------------------------------------------------
+
+#Internal Modules---------------------------------------------------------------
+from .PostProcessor import PostProcessor
+from . import validationAlgorithms
+from utils import utils, mathUtils
+from utils import InputData, InputTypes
+import DataObjects
+import MetricDistributor
+#Internal Modules End-----------------------------------------------------------
+
+
+class Validation(PostProcessor):
```

Comment:
please update the base class "PostProcessor" to "PostProcessorInterface"

Reply:
Done